cmake/utils.cmake

```diff
@@ -90,3 +90,57 @@ function(add_shaderc_tests)
       COMMAND ${TEST_NAME})
   endforeach()
 endfunction(add_shaderc_tests)
+
+# Finds all transitive static library dependencies of a given target.
+# This will skip ind libraries that were statically linked that were not
```

Comment:
Redundant `ind` here?
I'd also suggest revise this sentence. I find it a little awkward of having two `that`-clauses.